plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/controller-roles.yaml

```diff
@@ -907,6 +907,18 @@ items:
     verbs:
     - delete
     - list
+  - apiGroups:
+    - networking.k8s.io
+    resources:
+    - clustercidrconfigs
+    verbs:
+    - create
+    - delete
+    - get
+    - list
+    - patch
+    - update
```

Comment:
> 	// ClusterCIDRConfig Spec is immutable, no updates are allowed.

does the node-controller needs all these verbs?

Reply:
+1

Reply:
No doesn't need patch/update, removed the verbs

Reply:
maybe is my browser, I still can see the verbs :upside_down_face:

Reply:
I confirm it's just get and list now